components/Button/Button.tsx

```diff
@@ -20,13 +20,13 @@ const Button: FC<Props> = ({
   className,
 }) => {
   const styles = {
-    primary: '', // @todo: add styles when using these button types in ResourcesCards
+    primary: 'bg-primary-purple hover:bg-primary-purple/80 text-white rounded-[10px] duration-500', // @todo: add styles when using these button types in ResourcesCards
     secondary: '',
     pale: 'bg-slate-100 hover:bg-violet-50 dark:bg-white dark:bg-opacity-10 dark:hover:bg-opacity-20 text-slate-500 dark:text-violet-50',
     text: 'text-slate-400 dark:text-gray-400 !gap-4',
   }[variant]
 
-  const cls = clsx(
+  const cls = cn(
```

Comment:
Any specific reason for using `cn` instead of clsx?

Reply:
Yes.

Unlike `clsx`, `cn` is a utility function that merges both `clsx` and `tailwind-merge`. So, to ensure that the function isn't always written/repeated when writing new components, I wrote the function `cn` for this purpose, and I added it to `lib/utils.ts`, which is also a common pattern I see that is used in projects. A UI library like `Shadcn ` also uses the same pattern.